portal-frontend/src/app/features/applications/view-submission/view-application-submission.component.html

```diff
@@ -126,5 +114,47 @@ <h2>Applicant Submission</h2>
         </div>
       </mat-tab>
     </mat-tab-group>
+    <div *ngIf="application && application.canEdit" class="continue-button-container">
+      <button mat-stroked-button color="warn" (click)="onCancel(application.uuid)">
+        Cancel Application
+      </button>
+      <button mat-flat-button color="primary" *ngIf="application && application.canEdit" 
+        [routerLink]="'/application/' + application.fileNumber + '/edit'">
+        <div class="continue-button-content">
+          <span>Continue Application</span>
+          <mat-icon>keyboard_double_arrow_right</mat-icon>
+        </div>
+      </button>
+    </div>
+    <div *ngIf="application && application.canReview" class="continue-button-container">
+      <button mat-stroked-button color="warn" (click)="onCancel(application.uuid)">
+        Cancel Application
+      </button>
```

Comment:
This repeats the code from line 118-120, can you refactor the code to reduce code duplication?